bokeh/driving.py

```diff
@@ -59,12 +60,11 @@ def update(i):
     'repeat',
     'sine',
 )
-
 #-----------------------------------------------------------------------------
 # General API
 #-----------------------------------------------------------------------------
 
-def bounce(sequence):
+def bounce(sequence: Sequence[int]) -> "partial[Callable[[], None]]":
```

Comment:
Had to use a string here, otherwise the `-OO` test breaks, stating `partial` is not subscriptable. Offhand, would regard that as a bug in Python

Reply:
A side effect of cramming value and type level constructs into a single namespace.